File: app/src/objets/Exp.java
package objets;

public class Exp extends Drops {

    private int valeur;

    public Exp(int valeur) {
        this.valeur = valeur;
    }

    public int getValeur() {
        return valeur;
    }

    public void gainExp(Exp expGagnee) {
        valeur += expGagnee.getValeur();
    }

    public String toString() {
        return "+" + valeur + " XP";
    }
}

File: KF-Remake/Kingdom-Fall/source/objets/Exp.cs
namespace objets;

public class Exp : Drops
{
    internal int valeur { get; set; }

    public Exp(int valeur)
    {
        this.valeur = valeur;
    }

    public void GainXp(Exp expGagnee)
    {
        valeur += expGagnee.valeur;
    }

    public override string ToString()
    {
        return $"+{valeur} XP";
    }
}
